src/SFA.DAS.EAS.Web/Views/EmployerCommitments/RejectedTransferFundedCohorts.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@using SFA.DAS.EmployerCommitments.Web.ViewModels
+@model SFA.DAS.EmployerCommitments.Web.OrchestratorResponse<TransferFundedCohortsViewModel>
+
+@{
+    ViewBag.Title = "Rejected transfer requests";
+    ViewBag.Section = "apprentices";
+    ViewBag.PageId = "rejected-transfer-funded-cohorts";
+}
+
+<div class="grid-row">
+    <div class="column-full">
+        <h1 class="heading-xlarge">Rejected transfer requests</h1>
+
+        @{ var commitmentsCount = Model.Data.Commitments.Count(); }
+        <p class="dynamic-heading">You have <strong>@commitmentsCount</strong> cohort@(commitmentsCount > 1?"s":"") rejected transfer requests</p>
+
+        <table class="tableResponsive to-be-sent-list">
+            <thead>
+                <tr>
+                    <th>Sending employer</th>
+                    <th>Training provider</th>
+                    <th>Reference</th>
+                    <th></th>
+
+                </tr>
+            </thead>
+            <tbody>
+                @foreach (var commitment in Model.Data.Commitments)
+                {
+                <tr>
+                    <td>
+                        @commitment.SendingEmployer
+                    </td>
+                    <td>
+                        @commitment.ProviderName
+                    </td>
+                    <td>
+                        @commitment.HashedCommitmentId
+                    </td>
+                    <td>
+                        @switch (commitment.ShowLink)
```

Comment:
Maybe this came from the page you copied, but I'm not sure the "Details" link would ever show, would it? If it's been transfer-rejected, the user can always edit can't they? It might be worth keeping, however, since this approach allows the orchestrator to dictate the behaviour.